### ISSUE TITLE
TensorFlow 2.4.0, Python 3.8, CUDA 11, CuDNN 8, Compute 5.2, Ubuntu 20.04

### ISSUE THREAD

**Issue by marcossilva** [yaroslavvb/tensorflow-community-wheels#157]
- Ubuntu 20.04
- Tensorflow 2.4.0
- python 3.8.2
- CUDA 11.0
- CuDNN 8
- Compute Capabilities: 5.2

whl: 
https://github.com/marcossilva/tensorflow-cuda11-wheel/releases/download/v2.4.0/tensorflow-2.4.0-cp38-cp38-linux_x86_64.whl

More on:
https://github.com/marcossilva/tensorflow-cuda11-wheel

**Comment by wassname** [comment]
This is SSE4.2 as well :+1: , thanks
> 2020-08-04 10:10:00.204796: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE3 SSE4.1 SSE4.2

**Comment by shanhaidexiamo** [comment]
Is this GPU version?

**Comment by marcossilva** [comment]
@shanhaidexiamo Yes! It was built for the Compute Capability 5.2. You can find your GPU Compute Cabaility at <https://developer.nvidia.com/cuda-gpus>
